client/blocks/inline-help/inline-help-center-content.scss

```
@@ -318,6 +318,96 @@
 	}
 }
 
+/**
+* ARTICLE EMBED
+*/
+.help-center .help-center__container .inline-help__embed-result {
+	svg.gridicons-external {
+		height: 20px;
+		width: 20px;
+	}
+
+	h1.support-article-dialog__header-title-link {
+		color: var( --color-neutral-100 );
+		font-size: 20;
```

Adding `px` will make the bloody linter complain. 
```suggestion
		font-size: $font-title-small;
```